circle.yml

```diff
@@ -73,40 +71,48 @@ test:
   # Output the go versions we are going to test
     # - gvm use old && go version
     - gvm use stable && go version
-    # - gvm use bleed && go version
+    - gvm use bleed && go version
 
   # First thing: build everything. This will catch compile errors, and it's
   # also necessary for go vet to work properly (see #807).
     - gvm use stable && godep go install ./...:
         pwd: $BASE_STABLE
 
+    - gvm use bleed && godep go install ./...:
+        pwd: $BASE_BLEED
+
   # FMT
     - gvm use stable && test -z "$(gofmt -s -l . | grep -v Godeps/_workspace/src/ | tee /dev/stderr)":
         pwd: $BASE_STABLE
 
+    - gvm use bleed && test -z "$(gofmt -s -l . | grep -v Godeps/_workspace/src/ | tee /dev/stderr)":
+        pwd: $BASE_BLEED
+
   # VET
     - gvm use stable && go vet ./...:
         pwd: $BASE_STABLE
+    - gvm use bleed && go vet ./...:
+        pwd: $BASE_BLEED
 
   # LINT
     - gvm use stable && test -z "$(golint ./... | grep -v Godeps/_workspace/src/ | tee /dev/stderr)":
         pwd: $BASE_STABLE
 
-  override:
-  # Test every version we have (but stable)
-    # - gvm use old; godep go test -test.v -test.short ./...:
-    #     timeout: 600
-    #     pwd: $BASE_OLD
+    - gvm use bleed && test -z "$(golint ./... | grep -v Godeps/_workspace/src/ | tee /dev/stderr)":
+        pwd: $BASE_BLEED
 
-    # - gvm use bleed; go test -test.v -test.short ./...:
-    #     timeout: 600
-    #     pwd: $BASE_BLEED
+  override:
+     - gvm use bleed; go test -test.v -test.short ./...:
+         timeout: 600
+         pwd: $BASE_BLEED
 
   # Test stable, and report
   # Preset the goverall report file
-    - echo "$CIRCLE_PAIN" > ~/goverage.report
+  # - echo "$CIRCLE_PAIN" > ~/goverage.report
```

Comment:
You need to move that commented out section upper. It broke circle (questionable) parsing.